releasenotes/notes/fix_identity_operator_9e2ec9770ac046a6.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed a bug that caused :meth:`Statevector.expectation_value` to yield incorrect results
```

My bad, I missed the initial `.` here in my original suggestion -- otherwise this LGTM 🙂 
```suggestion
    Fixed a bug that caused :meth:`.Statevector.expectation_value` to yield incorrect results
```